ci/.travis.yml

```diff
@@ -1,142 +0,0 @@
-#
-# DO NOT EDIT THIS FILE
-#
-# It is automatically copied from https://github.com/pion/.goassets repository.
-# If this repository should have package specific CI config,
-# remove the repository name from .goassets/.github/workflows/assets-sync.yml.
-#
-# If you want to update the shared CI config, send a PR to
-# https://github.com/pion/.goassets instead of this repository.
-#
-
-dist: bionic
-language: go
-
-
-branches:
-  only:
-  - master
-
-env:
-  global:
-    - GO111MODULE=on
-    - GOLANGCI_LINT_VERSION=1.19.1
-
-cache:
-  directories:
-    - ${HOME}/.cache/go-build
-    - ${GOPATH}/pkg/mod
-  npm: true
-  yarn: true
-
-_lint_job: &lint_job
-  env: CACHE_NAME=lint
-  before_install:
-    - if [ -f .github/.ci.conf ]; then . .github/.ci.conf; fi
-  install: skip
-  before_script:
-    - |
-      curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh \
-        | bash -s - -b $GOPATH/bin v${GOLANGCI_LINT_VERSION}
-  script:
-    - bash .github/assert-contributors.sh
-    - bash .github/lint-disallowed-functions-in-library.sh
-    - bash .github/lint-commit-message.sh
-    - bash .github/lint-filename.sh
-    - golangci-lint run ${GOLANGCI_LINT_EXRA_ARGS:-} ./...
-_test_job: &test_job
-  env: CACHE_NAME=test
-  before_install:
-    - if [ -f .github/.ci.conf ]; then . .github/.ci.conf; fi
```

Comment:
As this wasn't ported to the GitHub Actions, 
https://github.com/pion/webrtc/blob/a33d7cdc37d3c0d865cefe6a0b6885f5d6eb7620/.github/.ci.conf#L1
is not used.
Because of this, `test.yaml` is not reusable among all repository at now.

https://github.com/pion/webrtc/pull/1889/files#diff-245392b692a50c38ecab4381b118862db514035c10983f3bd4f4b7f1f4be4692

@tarrencev @Sean-Der FYI